pkgs/package-overrides.nix

```
@@ -111,7 +111,7 @@ in
 
     ds = prev.extensions.ds.overrideAttrs (attrs: {
       preConfigure =
-        attrs.preConfigure or "" linkInternalDeps [ final.extensions.json ];
+        attrs.preConfigure or "" + linkInternalDeps [ final.extensions.json ];
```

I think we should make this

```suggestion
        let
          deps = lib.optionals (lib.versionOlder prev.php.version "8.0") [
            final.extensions.json
          ];
        in
        attrs.preConfigure or "" + linkInternalDeps deps;
```

Package sets should only contain packages and we should not have really been using `null` in the first place.

---

Fixed this everywhere. Good for you?